eng/mgmt/changelog/pom.xml

```diff
@@ -1,81 +1,125 @@
+                </exclusion>  
+                <exclusion>
+                    <groupId>org.glassfish.jaxb</groupId>
+                    <artifactId>jaxb-runtime</artifactId>
+                </exclusion>
+            </exclusions> 
+        </dependency>  
+        <dependency> 
+            <groupId>org.json</groupId>  
+            <artifactId>json</artifactId>  
+            <version>20201115</version> 
+        </dependency> 
+    </dependencies>  
+    <build> 
+        <plugins> 
+            <plugin> 
+                <groupId>org.codehaus.mojo</groupId>  
+                <artifactId>exec-maven-plugin</artifactId>  
+                <version>1.2.1</version>  
+                <executions> 
+                    <execution> 
+                        <goals> 
+                            <goal>java</goal> 
+                        </goals> 
+                    </execution> 
+                </executions>  
+                <configuration> 
+                    <mainClass>com.azure.resourcemanager.tools.changelog.Main</mainClass>  
+                    <systemProperties> 
+                        <systemProperty> 
+                            <key>PACKAGE_IGNORE</key>  
+                            <value>com\.azure\.resourcemanager(\.[^.]+)+\.fluent(\.[^.]+)*;com\.azure\.resourcemanager(\.[^.]+)+\.implementation</value> 
+                        </systemProperty>  
+                        <systemProperty> 
+                            <key>OLD_JAR</key>  
+                            <value>${OLD_JAR}</value>  
+                            <!-- Jar File Path --> 
+                        </systemProperty>  
+                        <systemProperty> 
+                            <key>NEW_JAR</key>  
+                            <value>${NEW_JAR}</value>  
+                            <!-- Jar File Path --> 
+                        </systemProperty> 
+                    </systemProperties> 
+                </configuration> 
+            </plugin>  
+            <plugin> 
+                <groupId>org.apache.maven.plugins</groupId>  
+                <artifactId>maven-compiler-plugin</artifactId>  
+                <version>3.5.1</version>  
+                <configuration> 
+                    <compilerArgs> 
+                        <arg>-Xlint:unchecked</arg> 
+                    </compilerArgs>  
+                    <source>${maven.compiler.source}</source>  
+                    <target>${maven.compiler.target}</target>  
+                    <encoding>UTF-8</encoding> 
+                </configuration> 
+            </plugin> 
+        </plugins> 
+    </build>  
+    <dependencyManagement> 
+        <dependencies> 
+            <dependency> 
+                <groupId>org.jvnet.staxex</groupId>  
+                <artifactId>stax-ex</artifactId>  
+                <version>1.8.1</version> 
+            </dependency> 
+        </dependencies> 
+    </dependencyManagement> 
```

Comment:
What is the purpose for this dependency?

Reply:
@ChenTanyi Thanks for your feedback.
**_org.jvnet.staxex 1.8.1_** is transitively introduced by **_org.glassfish.jaxb:jaxb-runtime:jar_**.
**_org.glassfish.jaxb:jaxb-runtime:jar_** is not used by **_azure-sdk-for-java_**.
However, because of a dependency conflict issue, **_org.jvnet.staxex 1.8.1_** is referenced by your project.
As such, the **_dependency management_** tag can help to fix a version of **_org.jvnet.staxex_**, to address such a dependency conflict.
If you do not like the **_dependency management_** tag, please ignore it, which would affect the program behaviors.

I can submit a new PR without the  **_dependency management_** tag, and use 2 space indent.